service_mesh/service_mesh_install/ossm-jaeger.adoc

```diff
@@ -0,0 +1,26 @@
+[id="ossm-jaeger"]
+= Jaeger overview
+include::modules/ossm-document-attributes.adoc[]
+:context: jaeger
+
+toc::[]
+
+Every time a user takes an action in an application, a request is executed by the architecture that may require dozens of different services to participate in order to produce a response.
+The path of this request is a distributed transaction.
+Jaeger lets you perform distributed tracing, which follows the path of a request through various microservices that make up an application.
+
+*Distributed tracing* is a technique that is used to tie the information about different units of work together—usually executed in different processes or hosts—in order to understand a whole chain of events in a distributed transaction.
+Distributed tracing lets developers visualize call flows in large service oriented architectures.
+It can be invaluable in understanding serialization, parallelism, and sources of latency.
+
+Jaeger records the execution of individual requests across the whole stack of microservices, and presents them as traces. A *trace* is a data/execution path through the system. An end-to-end trace is comprised of one or more spans.
+
+A *span* represents a logical unit of work in Jaeger that has an operation name, the start time of the operation, and the duration. Spans may be nested and ordered to model causal relationships.
```

Comment:
I think you are aware, but spans might also contain other information, like tags and logs. I agree with leaving them out in case you left them out in the name of simplicity.